README.md

```diff
@@ -36,10 +36,11 @@ To enable debugging the JSON output in a browser, adding the query parameter `ac
 
 - Download Docker
 - Navigate to the root directory of this project
-- Run `docker build --tag 'diplicity' .`
-- Run `docker run -p 8080:8080 -p 8000:8000 diplicity`
+- Run `docker-compose up`
 - The API is now available on your machine at `localhost:8080`
 - The Admin server is now available on your machine at `localhost:8000`
+- **Note** to get the Discord bot auth to work, you need to send a curl request
+  after the service is initialized: `curl -XPOST http://localhost:8080/_configure -d '{"DiscordBotCredentials": {"Username": "<username>", "Password": "<password>"}}'`
```

Comment:
Do you need both username and password? We won't have multiple bots with different purposes?

Reply:
Probably not. I was just following Basic auth standard. I don't think it matters much either way